src/content-switcher/content-switcher-option.directive.ts

```diff
@@ -58,7 +58,7 @@ export class ContentSwitcherOption {
 
 	@HostListener("blur", ["$event"])
 	onBlur(event) {
-		if (event.relatedTarget) {
+		if (event.relatedTarget && event.relatedTarget.className === "bx--content-switcher-btn") {
```

Comment:
Since this is a directive, we should prefer `classList.contains` ... I don't expect it will be needed, but we _should_ support cases where we have additional classes applied.

```suggestion
		if (event.relatedTarget && event.relatedTarget.classList.contains("bx--content-switcher-btn")) {
```